.lintstagedrc

```diff
@@ -0,0 +1,3 @@
+{
+    "*.php": "phplint"
```

Comment:
If necessary, we can add linting commands for other files as well.

The only requirement is that the linter should accept the list of the modified files.

For example, for `phplint`, the actual command will be

```bash
phplint /path/to/changed/file1.php /path/to/changed/file2.php /path/to/changed/file3.php
```

This is the reason why we cannot use `php -l`: it will check only the first passed file, and treat the rest as command line parameters to the first file.